src/main/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/SugarRemovalUtilityFragmenter.java

```diff
@@ -235,7 +237,8 @@ public static enum SRUFragmenterReturnedFragmentsOption {
     public SugarRemovalUtilityFragmenter() {
         this.sugarRUInstance = new SugarRemovalUtility(SilentChemObjectBuilder.getInstance());
         this.settings = new ArrayList<>(15);
```

Comment:
Where does this magic number come from?

Reply:
Again, the number of settings. I have declared it accordingly now.